.github/workflows/coverage.yml

```diff
@@ -44,6 +45,7 @@ jobs:
     - uses: actions/checkout@v3
       with:
         fetch-depth: '0'
+        persist-credentials: false
```

Comment:
What is the intent of this change?

Reply:
Oh, I added that because I was fetching the `gcovr` repo as the fix was not yet released. It's not needed anymore, so I'll remove it.